<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
  <class>QrCodeScanWidget</class>
  <widget class="QWidget" name="QrCodeScanWidget">
    <property name="geometry">
      <rect>
	<x>0</x>
	<y>0</y>
	<width>526</width>
	<height>406</height>
      </rect>
    </property>
    <property name="windowTitle">
      <string>Form</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
      <property name="leftMargin">
	<number>0</number>
      </property>
      <property name="topMargin">
	<number>0</number>
      </property>
      <property name="rightMargin">
	<number>0</number>
      </property>
      <property name="bottomMargin">
	<number>0</number>
      </property>
      <item>
	<layout class="QHBoxLayout" name="horizontalLayout_2">
	  <item>
	    <widget class="QLabel" name="label_3">
	      <property name="sizePolicy">
		<sizepolicy hsizetype="Maximum" vsizetype="Preferred">
		  <horstretch>0</horstretch>
		  <verstretch>0</verstretch>
		</sizepolicy>
	      </property>
	      <property name="text">
		<string>Camera:</string>
	      </property>
	    </widget>
	  </item>
	  <item>
	    <widget class="QComboBox" name="combo_camera">
	      <property name="sizePolicy">
		<sizepolicy hsizetype="Preferred" vsizetype="Fixed">
		  <horstretch>0</horstretch>
		  <verstretch>0</verstretch>
		</sizepolicy>
	      </property>
	    </widget>
	  </item>
	  <item>
	    <widget class="QPushButton" name="btn_refresh">
	      <property name="sizePolicy">
		<sizepolicy hsizetype="Maximum" vsizetype="Fixed">
		  <horstretch>0</horstretch>
		  <verstretch>0</verstretch>
		</sizepolicy>
	      </property>
	      <property name="text">
		<string>Refresh</string>
	      </property>
	    </widget>
	  </item>
	  <item>
	    <spacer name="horizontalSpacer">
	      <property name="orientation">
		<enum>Qt::Horizontal</enum>
	      </property>
	      <property name="sizeHint" stdset="0">
		<size>
		  <width>40</width>
		  <height>20</height>
		</size>
	      </property>
	    </spacer>
	  </item>
	</layout>
      </item>
      <item>
	<widget class="InfoFrame" name="frame_error">
	  <property name="frameShape">
	    <enum>QFrame::StyledPanel</enum>
	  </property>
	  <property name="frameShadow">
	    <enum>QFrame::Raised</enum>
	  </property>
	</widget>
      </item>
      <item>
	<widget class="QVideoWidget" name="viewfinder" native="true">
	  <property name="sizePolicy">
	    <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
	      <horstretch>0</horstretch>
	      <verstretch>0</verstretch>
	    </sizepolicy>
	  </property>
	</widget>
      </item>
      <item>
	<widget class="QProgressBar" name="progressBar_UR">
	  <property name="maximum">
	    <number>1</number>
	  </property>
	  <property name="value">
	    <number>0</number>
	  </property>
	  <property name="format">
	    <string>%v / %m</string>
	  </property>
	</widget>
      </item>
      <item>
	<layout class="QHBoxLayout" name="horizontalLayout">
	  <item>
	    <widget class="QCheckBox" name="check_manualExposure">
	      <property name="text">
		<string>Manual exposure</string>
	      </property>
	    </widget>
	  </item>
	  <item>
	    <widget class="QSlider" name="slider_exposure">
	      <property name="minimum">
		<number>1</number>
	      </property>
	      <property name="maximum">
		<number>100</number>
	      </property>
	      <property name="pageStep">
		<number>1</number>
	      </property>
	      <property name="orientation">
		<enum>Qt::Horizontal</enum>
	      </property>
	    </widget>
	  </item>
	</layout>
      </item>
    </layout>
    <zorder>progressBar_UR</zorder>
    <zorder></zorder>
    <zorder>viewfinder</zorder>
    <zorder>frame_error</zorder>
    <zorder>horizontalLayoutWidget</zorder>
  </widget>
  <customwidgets>
    <customwidget>
      <class>InfoFrame</class>
      <extends>QFrame</extends>
      <header>components.h</header>
      <container>1</container>
    </customwidget>
    <customwidget>
      <class>QVideoWidget</class>
      <extends>QWidget</extends>
      <header>qvideowidget.h</header>
      <container>1</container>
    </customwidget>
  </customwidgets>
  <resources/>
  <connections/>
</ui>
